docs/yuidoc-p5-theme/layouts/main.handlebars

```diff
@@ -38,16 +38,22 @@
     <!--<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>-->
     <script>window.jQuery || document.write('{{p5SiteRoot}}/js/vendor/jquery-1.9.1.min.js"><\/script>')</script>
 
+    <!-- underscore-->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
+
+    <!-- backbone-->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/backbone.js/1.3.3/backbone-min.js"></script>
```

Comment:
@lmccart Would something like this work with the website or would it create duplicates?